packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -305,18 +295,17 @@ The `dns` section specifies configuration options for the DNS protocol. The DNS
 
 [source,yaml]
 ------------------------------------------------------------------------------
-packetbeat.protocols:
-  dns:
-    ports: [53]
+packetbeat.protocols.dns:
+  ports: [53]
 
-    # include_authorities controls whether or not the dns.authorities field
-    # (authority resource records) is added to messages.
-    # Default: false
-    include_authorities: true
-    # include_additionals controls whether or not the dns.additionals field
-    # (additional resource records) is added to messages.
-    # Default: false
-    include_additionals: true
+  # include_authorities controls whether or not the dns.authorities field
```

Comment:
General comment/question: We're a bit inconsistent about including comment blocks in example configs. I'm not saying that we have to be consistent, but it seems a bit arbitrary sometimes. Curious to know if other reviewers think that we should standardize on an approach.